hrpc/compact.go

```diff
@@ -0,0 +1,57 @@
+package hrpc
+
+import (
+	"context"
+	"strings"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/tsuna/gohbase/pb"
+)
+
+type CompactRegion struct {
+	tableOp
+
+	region string
+	major  bool
+	family []byte
+}
+
+//Create a new CompactRegion struct
+func NewCompactRegion(region string, family []byte, major bool) *CompactRegion {
```

Comment:
you should provide the context as the first argument.
also, is there a reason for "region" being a string and "family" being []byte?